admin-auth/src/main/kotlin/com/epam/drill/admin/auth/config/ApiKeyAuthenticationProvider.kt

```diff
@@ -56,6 +62,11 @@ class ApiKeyAuthenticationProvider internal constructor(
          */
         var headerName: String = API_KEY_HEADER
 
+        /**
+         * The size that will be used for the cache.
```

Comment:
This doesn't explain how to use it. Not by whom it is used, but how or for what

Reply:
rewrited